src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1932,7 +1932,7 @@ class AccessStatementSegment(BaseSegment):
             ),
         ),
         Sequence("APPLY", "MASKING", "POLICY"),
-        Sequence("APPLY", "RAW", "ACCESS", "POLICY"),
+        Sequence("APPLY", "ROW", "ACCESS", "POLICY"),
```

Comment:
Maybe add a test case for this (and ideally others)?

As penance! 😉 And mroe seriously to catch any future regressions.

Reply:
Sorry for that! I'll add tests now

Reply:
I added tests for all cases I added yesterday. I'm very sorry for the typo I've made. I immediately made a new PR to fix it when I discovered this problem.

Reply:
All good - no need to apologise!